### DISCUSSION TITLE
Firefly III importer error

### DISCUSSION THREAD

**Discussion by motionthings** [firefly-iii/firefly-iii#6185] [Firefly III] [answered]
What am I doing wrong here?

Trying to import a csv file that looks likt this:

![import-csv](https://user-images.githubusercontent.com/264911/175001252-742f29a3-da8f-4bf8-9119-02acf408a1b8.png)

The fields translate as the following:

Date, Description, Interest date, amount (credit), amount (debit), to account, from account.

First config page page:

![import-config-1](https://user-images.githubusercontent.com/264911/175001366-32d17583-5cca-4b29-92d3-7f422c126a74.png)

Second config page:

![role definition](https://user-images.githubusercontent.com/264911/175001463-3e0cda73-636a-4a9b-ab57-95ca3b238a5a.png)

I also looked at the import configuration for my bank here:
https://github.com/firefly-iii/import-configurations/blob/main/no/sparebank1/default.json
This gives me a server error 500
It does however look correct to me.

When I try an import with the above settings it imports (creates) all withdrawals but not deposits.
And if I do the accounts the other way around it creates deposits but not withdraws.

![import](https://user-images.githubusercontent.com/264911/175001559-e38537b7-9de7-4e70-8582-e666c0b4a5b5.png)

I am using the docker version here: https://hub.docker.com/r/fireflyiii/data-importer
'docker pull fireflyiii/data-importer:latest'

I am now going to try the dev version 'docker pull fireflyiii/data-importer:develop'

Am I doing something obviously wrong or is this a bug?

Simon

**Comment by JC5** [comment]
Could you share one or two lines and your config file?

**Comment by motionthings** [comment, marked as answer]
Yep. I'm dumb. lol
I right clicked and chose "save file as..." in the github repository on the json file for my bank.
That gives you a html file with a .json extension.

With a proper json file for my bank I also understood my mistake.
Both the "to account" and "from account" are asset accounts and not opposing accounts.

Thank you for making me look inside the json file :)

Simon

**Reply by JC5** [reply]
Haha glad to hear it works!

**Reply by motionthings** [reply]
Yup. I have now imported 6000 transactions (10 years) and tested Nordigen import too.
Everything works just fine.
Now I "only" have to categorize 6000+ transactions :)
